lib/nero/wildlife_watch/wildlife_watch_classification.rb

```diff
@@ -22,7 +22,7 @@ def annotations
 
       def task
         case workflow_id
-        when "1021"
+        when "1021", "1717"
```

Comment:
I'm going to refactor this into something generic later this week. But they've got a launch coming tomorrow and I wanted to get it out first.